### ISSUE TITLE
training after an epoch initial checpoint still from pretrained model 

### ISSUE THREAD

**Issue by LiuPearl1** [rishizek/tensorflow-deeplab-v3#22]
I have succeeded in training. After one epoch, the model restore and Initialize variable. But I find the variable were initialized from pretrained model. Does this mean that the variable in ResNet are not changed after training?  

Like this:
`learning_rate = 0.006867579, cross_entropy = 0.716586, train_px_accuracy = 0.81120086, train_mean_iou = 0.3955025 (17.789 sec)
INFO:tensorflow:learning_rate = 0.006865475, cross_entropy = 0.43522605, train_px_accuracy = 0.8115431, train_mean_iou = 0.3969977 (17.650 sec)
INFO:tensorflow:learning_rate = 0.0068633705, cross_entropy = 0.40337503, train_px_accuracy = 0.8122774, train_mean_iou = 0.3994764 (17.608 sec)
INFO:tensorflow:learning_rate = 0.006861266, cross_entropy = 0.2647943, train_px_accuracy = 0.8138005, train_mean_iou = 0.40384638 (17.601 sec)
INFO:tensorflow:Saving checkpoints for 662 into ./model/aspp_dense_attention/model.ckpt.
INFO:tensorflow:Loss for final step: 11.729751.
INFO:tensorflow:Start training.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Initialize variable resnet_v2_101/block1/unit_1/bottleneck_v2/conv1/BatchNorm/beta:0 from checkpoint ./ini_checkpoints/resnet_v2_101/resnet_v2_101.ckpt with resnet_v2_101/block1/unit_1/bottleneck_v2/conv1/BatchNorm/beta
INFO:tensorflow:Initialize variable resnet_v2_101/block1/unit_1/bottleneck_v2/conv1/BatchNorm/gamma:0 from checkpoint ./ini_checkpoints/resnet_v2_101/resnet_v2_101.ckpt with resnet_v2_101/block1/unit_1/bottleneck_v2/conv1/BatchNorm/gamma
INFO:tensorflow:Initialize variable resnet_v2_101/block1/unit_1/bottleneck_v2/conv1/BatchNorm/moving_mean:0 from checkpoint ./ini_checkpoints/resnet_v2_101/resnet_v2_101.ckpt with resnet_v2_101/block1/unit_1/bottleneck_v2/conv1/BatchNorm/moving_mean...........`

Can you explain it?
